grails-app/controllers/au/org/ala/profile/api/CollectionList.groovy

```diff
@@ -1,4 +1,4 @@
-package au.org.ala.profile.domain
+package au.org.ala.profile.api
```

Comment:
The location is not correct. Should not place this file with controllers. See comment before to get exact location.